src/Enum.php

```diff
@@ -188,6 +188,13 @@ public static function toArray(): array
         return array_combine(static::getValues(), static::getIndices());
     }
 
+    public static function getAll(): array
+    {
+        return array_map(function(string $name) {
+            return static::$name();
+        }, static::getNames());
```

Comment:
It would be faster to resolve by index - this is the first case in `make()` and also a clear resolve function because it doesn't have to decide if it's a value or name.

Reply:
should be done in the following `v2` PR

Reply:
Done

Reply:
https://github.com/spatie/enum/pull/33/commits/051fa365f34e2c25b1d28c9a5e85ca628815e86c